llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -1084,6 +1091,7 @@ bool VPInstruction::opcodeMayReadOrWriteFromMemory() const {
   case VPInstruction::LogicalAnd:
   case VPInstruction::Not:
   case VPInstruction::PtrAdd:
+  case VPInstruction::WidePtrAdd:
   case VPInstruction::WideIVStep:
```

Comment:
```suggestion
  case VPInstruction::WideIVStep:
  case VPInstruction::WidePtrAdd:
```

lex order (even though its not consistent throughout)